### DISCUSSION TITLE
The ‘growth’ program is not compatible

### DISCUSSION THREAD

**Discussion by siyu7992** [SysBioChalmers/yeast-GEM#365] [HELP!]
run R2 = growth(model) in matlab, model .mat and .xml both worked correctly, but the .xml format failed :
Invalid coefficients defined for reaction(s): r_4047, r_4048, r_4049.

**Comment by edkerk** [comment]
Could you please provide all the commands that you used, to load the model versions etc., to replicate this?

Also, could you please specify the model and RAVEN versions?

**Reply by siyu7992** [reply]
model = readCbModel('yeast-GEM.xml');
setRavenSolver('cobra')
R2 = growth(model);

and RAVEN2.8.4  model yeast-GEM9.0.0

**Reply by edkerk** [reply]
The `growth` function assumes that the model is loaded as RAVEN format. So you should either use the `loadYeastModel` function as described in the [README.md](https://github.com/SysBioChalmers/yeast-GEM?tab=readme-ov-file#model-usage), or directly use RAVEN's `importModel` function.
